examples/cc-template/flake.lock

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
This needs an update. I'm getting changes to this file when I run `nix develop`.